README.md

```diff
@@ -36,9 +37,10 @@ During help request workflow the application:
 
 1. Asks Azure AI services for recommendation for area, environment and team.
 2. Searches Azure AI search for similar requests.
```

Comment:
Just wondering - will this search for currently open requests or just closed? if a team flood the help-bot with multiple requests

Reply:
all requests I think although status is stored in cosmos db (although I don't think i'm syncing it atm, but I stored it just in case we want to in the future)